libsolidity/ast/Types.cpp

```diff
@@ -110,6 +110,14 @@ util::Result<TypePointers> transformParametersToExternal(TypePointers const& _pa
 	return transformed;
 }
 
+string typesToString(TypePointers const& _types, bool _short)
```

Comment:
I think that the name of this function is too generic for what it does. It specifically creates a parameters for use in a signature and makes assumptions related to that, i.e. that there should be no space after `,` or that you need the parentheses. From the name I would assume that I could use it for example to print the parameter list for a function definition but that's not the case (you need the space). So I'd change it like this:
```suggestion
string formatSignatureParameterList(TypePointers const& _types)
```
Then assume also that `_short` is always `true`.

Reply:
Name updated, but I think we still need `bool _short` in order to reuse this in `FunctionType::toString`?

Reply:
Right. You do need `_short`. I missed the fact that in the `toString()` we do use it.